seqr/views/apis/dashboard_api_tests.py

```diff
@@ -120,3 +122,52 @@ def test_export_projects_table(self):
         response = self.client.get('{}?file_format=csv'.format(url))
         self.assertEqual(response.status_code, 400)
         self.assertDictEqual(response.json(), {'error': 'Invalid file_format: csv'})
+
+
+# Tests for AnVIL access disabled
+class LocalDashboardPageTest(AuthenticationTestCase, DashboardPageTest):
+    fixtures = ['users', '1kg_project']
+    NUM_COLLABORATOR_PROJECTS = 2
+    NUM_STAFF_PROJECTS = 3
+    NUM_STAFF_EXPORT_RECS = 3
+    NUM_STAFF_EXPORT_TSV_LINES = NUM_STAFF_EXPORT_RECS + 1
+
+
+# Test for permissions from AnVIL only
+class AnvilDashboardPageTest(AnvilAuthenticationTestCase, DashboardPageTest):
+    fixtures = ['users', 'social_auth', '1kg_project']
+    NUM_COLLABORATOR_PROJECTS = 1
+    NUM_STAFF_PROJECTS = 3
+    NUM_STAFF_EXPORT_RECS = 3
+    NUM_STAFF_EXPORT_TSV_LINES = NUM_STAFF_EXPORT_RECS + 1
+
```

Comment:
this class needs to also test that the terra calls were called correctly, similar to what you do on the mix test case

Reply:
Okay